File: Java/src/Student.java
/** Represents one student, their exam group, and whether they are a listed cheater. */
public class Student {
    int id;
    int groupId;
    boolean cheater;

    public Student(int ID, int groupId) {
        this.id = ID;
        this.groupId = groupId;
        cheater = false;
    }

    public int getId() {
        return this.id;
    }

    public int getGroupId() {
        return this.groupId;
    }

    public boolean isCheater() {
        return cheater;
    }
}

File: python/src/Student.py
class Student:
    """Represents one student in the exam distribution problem."""

    def __init__(self, student_id: int, group_id: int):
        self.id = student_id
        self.group_id = group_id
        self.cheater = False

    def __repr__(self) -> str:
        return f"Student(id={self.id}, group_id={self.group_id}, cheater={self.cheater})"
